Sources/EpoxyCore/SwiftUI/EpoxyableView+SwiftUIView.swift

```diff
@@ -0,0 +1,296 @@
+      wrapper.uiView.setBehaviors(behaviors)
+    }
+
+    // We need to create a new view instance when the style is updated.
+    guard wrapper.view.style == style else {
+      let uiView = View(style: style)
+      uiView.setBehaviors(behaviors)
+      wrapper.uiView = uiView
+      return
+    }
+
+    // No updates required.
+  }
+
+  func makeUIView(context: Context) -> EpoxyableViewContainer<Self, View> {
+    let uiView = View(style: style)
+    uiView.setBehaviors(behaviors)
+    return EpoxyableViewContainer(view: self, uiView: uiView)
+  }
+}
+
+// MARK: - SwiftUIStylelessContentlessEpoxyableView
+
+/// A SwiftUI `View` representing an `EpoxyableView` with a `Never` `Style` and `Content`.
+struct SwiftUIStylelessContentlessEpoxyableView<View: EpoxyableView>: UIViewRepresentable {
+
+  // MARK: Lifecycle
+
+  init(behaviors: View.Behaviors? = nil) {
+    self.behaviors = behaviors
+  }
+
+  // MARK: Internal
+
+  var behaviors: View.Behaviors?
+
+  func updateUIView(_ wrapper: EpoxyableViewContainer<Self, View>, context: Context) {
+    wrapper.view = self
+    wrapper.uiView.setBehaviors(behaviors)
+  }
+
+  func makeUIView(context: Context) -> EpoxyableViewContainer<Self, View> {
+    let uiView = View()
+    uiView.setBehaviors(behaviors)
+    return EpoxyableViewContainer(view: self, uiView: uiView)
+  }
+}
+
+// MARK: - EpoxyableViewContainer
+
+/// A view that has an `intrinsicContentSize` of the `view`'s `systemLayoutSizeFitting(…)`.
+final class EpoxyableViewContainer<SwiftUIView: UIViewRepresentable, UIViewType: EpoxyableView>: UIView {
+
+  // MARK: Lifecycle
+
+  init(view: SwiftUIView, uiView: UIViewType) {
+    self.view = view
+    self.uiView = uiView
+
+    super.init(frame: .zero)
+
+    addSubview(uiView)
+    setUpConstraints()
+  }
+
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: Internal
+
+  var view: SwiftUIView
+
+  var uiView: UIViewType {
+    didSet { updateView(from: oldValue) }
+  }
+
+  override var intrinsicContentSize: CGSize {
+    uiView.systemLayoutSizeFitting(
+      UIViewType.layoutFittingCompressedSize,
+      withHorizontalFittingPriority: .defaultLow,
+      verticalFittingPriority: .defaultLow)
+  }
+
+  // MARK: Private
+
+  private func updateView(from oldValue: UIViewType) {
+    guard uiView !== oldValue else { return }
+    oldValue.removeFromSuperview()
+    addSubview(uiView)
+    setUpConstraints()
+    invalidateIntrinsicContentSize()
+  }
+
+  private func setUpConstraints() {
+    uiView.translatesAutoresizingMaskIntoConstraints = false
+
+    let leading = uiView.leadingAnchor.constraint(equalTo: leadingAnchor)
+    let top = uiView.topAnchor.constraint(equalTo: topAnchor)
+
+    let trailing = uiView.trailingAnchor.constraint(equalTo: trailingAnchor)
+    trailing.priority = .defaultHigh
+
+    let bottom = uiView.bottomAnchor.constraint(equalTo: bottomAnchor)
+    bottom.priority = .defaultHigh
```

Comment:
The bottom constraint makes sense to me, but we almost always do required priority for leading, top, and trailing. Why did you decide to only use default high for trailing here?